tests/FSharp.Compiler.Private.Scripting.UnitTests/DependencyManagerInteractiveTests.fs

```diff
@@ -232,7 +230,7 @@ type DependencyManagerInteractiveTests() =
 
 /// Native dll resolution is not implemented on desktop
 #if NETCOREAPP
-    [<Fact>]
+    [<Fact(Skip="downloads very large ephemeral packages"); >]
```

Comment:
Should this test even exist right now? It feels like the packages there are undergoing so much churn that by the time we ever consider unskipping everything will be different

Reply:
I guess we could pin the packages and they are on nuget.org forever